### PR TITLE
feat/#109 enhance unit test

### PR DESCRIPTION
**Pull request by boomspeed94**
#109

### CODE REVIEW

File: packages/demux-icon/src/IconActionReader.ts [hunk 1]
```diff
@@ -76,7 +80,7 @@ export class IconActionReader extends AbstractActionReader {
 
       return irreversibleBlockNum;
     } catch (err) {
-      this.log.error(err);
+      // this.log.error(err);
```
Comment (manh-vv) [inline packages/demux-icon/src/IconActionReader.ts:83]: @boomspeed94 I don't see any problem with this error log. Please do not comment it out.

File: packages/demux-icon/src/IconActionReader.ts [hunk 2]
```diff
@@ -102,7 +106,7 @@ export class IconActionReader extends AbstractActionReader {
 
       return block;
     } catch (err) {
-      this.log.error(err);
```
Comment (manh-vv) [inline packages/demux-icon/src/IconActionReader.ts:105]: same here

**Review by manh-vv — CHANGES_REQUESTED** [review]
@boomspeed94 there only two error logs I would like you to keep as it is. Other things look good to me.

File: packages/demux-icon/src/IconActionReader.ts (hunk 1 above)
Comment (boomspeed94) [reply packages/demux-icon/src/IconActionReader.ts]: @manh-vv  reverted

File: packages/demux-icon/src/IconActionReader.ts (hunk 2 above)
Comment (boomspeed94) [reply packages/demux-icon/src/IconActionReader.ts]: @manh-vv reverted https://github.com/lecle/aloxide/pull/110/commits/1d2c5bf4760f45e3a7b3198cd6cf744c34825779

**Review by manh-vv — APPROVED** [review]
LGTM

**Pull request merged**